pkg/ttnpb/field_mask_validation.go

```diff
@@ -553,7 +553,21 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 	"/ttn.lorawan.v3.EntityRegistrySearch/SearchOrganizations": OrganizationFieldPathsNested,
 
 	// Users:
-	"/ttn.lorawan.v3.UserRegistry/Get":                 UserFieldPathsNested,
-	"/ttn.lorawan.v3.UserRegistry/Update":              UserFieldPathsNested,
-	"/ttn.lorawan.v3.EntityRegistrySearch/SearchUsers": UserFieldPathsNested,
+	"/ttn.lorawan.v3.UserRegistry/Get":                 omitFields(UserFieldPathsNested, "password", "temporary_password"),
+	"/ttn.lorawan.v3.UserRegistry/Update":              omitFields(UserFieldPathsNested, "password", "temporary_password"),
```

Comment:
Shouldnt' the `password` be allowed here? Or does the stack generate it?

Reply:
There is a separate RPC for changing the password